submissions/andysmokk/oop-exercise/index.js

```diff
@@ -0,0 +1,118 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/andysmokk/a-tiny-JS-world
+   Web app: https://andysmokk.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+  constructor(species, name, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends.join(", ");
+  }
+
+  showPropertiesInhabitant() {
+    const valuesInhabitant = [];
+    const propertiesInhabitant = [
+      "species",
+      "name",
+      "gender",
+      "saying",
+      "friends",
+    ];
+
+    propertiesInhabitant.forEach((property) => {
+      property === "name"
+        ? valuesInhabitant.push(`<strong>${this[property]}</strong>`)
+        : property === "saying"
+        ? valuesInhabitant.push(`<em>"${this[property]}"</em>`)
+        : valuesInhabitant.push(this[property]);
+    });
+
+    return valuesInhabitant;
```

Comment:
(1) This is a classic use-case for `Array#map`

(2) Why not compose `propertiesInhabitant` of property values instead of string names?

(3) Did you know you can put any valid JS experession as an array's element?
See the example below:
```
[
  'some string',
  this.property,
  `<strong>${variable}</strong>`,
  this.arrProperty.join(", "),
]
```

Reply:
Fixed.